eslint/babel-eslint-parser/src/worker/index.cts

```diff
@@ -0,0 +1,31 @@
+import babel = require("./babel-core.cts");
+import handleMessage = require("./handle-message.cts");
+
+import worker_threads = require("worker_threads");
```

Comment:
Is `import` always necessary, or given that we are not using any _type_ exported by `worker_threads` can we use `const` here?

Reply:
It should be unnecessary, but the `import` will be more uniform?